src/main/java/org/marek/model/User.java

```diff
@@ -0,0 +1,18 @@
+package org.marek.model;
+
+import javax.persistence.Entity;
+
+@Entity
+
+public class User {
+    private Integer id;
+    private String name;
+    private String surname;
+    private Integer phoneNumber;
+    private String mail;
```

Comment:
Bude povinné, proto zvolíme anotaci @NotNull. Taky opět zjistit `@Size(min=X, max=Y)` minimální hodnotu a zkusit vytvořit nějakou rozumnou max hodnotu.